src/argilla/server/contexts/accounts.py

```diff
@@ -43,6 +49,13 @@ def create_user(db: Session, user_create: UserCreate):
     return user
 
 
+def delete_user(db: Session, user: User):
+    db.delete(user)
```

Comment:
Removing this user from the user_workspaces relationship table will be tackled in a next step, right?

Reply:
That's correct.